common/app/common/pages_helpers.cljc

```diff
@@ -58,7 +58,7 @@
 (defn get-object-with-children
   "Retrieve a list with an object and all of its children"
   [id objects]
-  (map #(get objects %) (concat [id] (get-children id objects))))
+  (map #(get objects %) (d/concat [id] (get-children id objects))))
```

Comment:
**minor:**  for a single element concat on start when the result type is irrelevant (this case, bacause you then using `map`, that returns a seq), the best approach is use `cons` (example: `(cons id (get-children id objects))`)

Reply:
fixed